docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,13 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use in the propagation
+stage. These can be anything from default or weak passwords, to "stolen" credentials from your
```

Comment:
I don't think "propagation stage" is a thing in our tool. Maybe it's better to say:
 "On this screen, you can configure credentials that brute-force exploiters will attempt.".

Reply:
Another option would be "during propagation". "You can configure credentials for Infection Monkey to use during propagation."